Python/Tests/Analysis/AnalysisTest.Perf.cs

```diff
@@ -325,7 +325,7 @@ internal sealed class FileStreamContentProvider : StreamContentProvider {
 
         private static void CollectFiles(string dir, List<string> files, ISet<string> excludeDirectories = null) {
             foreach (string file in Directory.GetFiles(dir)) {
-                if (file.EndsWith(".py", StringComparison.OrdinalIgnoreCase)) {
+                if (file.EndsWithOrdinal(".py", ignoreCase: true)) {
```

Comment:
`EndsWithOrdinal`  with parameter or `EndsWithOrdinalIgnoreCase`?

Reply:
The argument matches other string functions that don't take a culture info object.